tests/v3_api/test_auth.py

```diff
@@ -108,6 +108,32 @@ def test_ad_service_account_login():
         login(SERVICE_ACCOUNT_NAME, SERVICE_ACCOUNT_PASSWORD)
 
 
+def special_character_users_login(access_mode):
+    delete_project_users()
+    delete_cluster_users()
+    auth_setup_data = setup["auth_setup_data"]
+    admin_user = auth_setup_data["admin_user"]
+    admin_token = login(admin_user, PASSWORD)
```

Comment:
yes, before this we did not disable AD, so we assume AD is already enabled. Since we only calling disable AD in this function, not the previous sequential test functions